superstitious.yml

```diff
@@ -0,0 +1,48 @@
+# Superstitious GitHub Configuration
+# Configure which numbers to avoid and behavior settings
+
+# Numbers to avoid for issues and pull requests
+unlucky_numbers:
+  - 7
+  - 13
+  - 66
+  - 77
+  - 666
+  - 777
+  - 1313
+  - 1337
+
+# How many numbers ahead to reserve to prevent race conditions
+# when multiple issues/PRs are created simultaneously
+reservation_space: 5
+
+# Whether to enable clearing mode by default
+# In clearing mode, existing unlucky issues/PRs are duplicated
+# with safe numbers and the original unlucky ones are deleted
+clearing_mode: false
+
+# Placeholder issue settings
+placeholder:
+  title: "🔮 Reserved for superstitious purposes"
+  body: |
+    This issue was created automatically to prevent an unlucky number from being assigned.
+    
+    **This is a placeholder issue and will be deleted shortly.**
+    
+    > Unlucky numbers avoided: 7, 13, 66, 77, 666, 777, 1313, 1337
+  labels:
+    - "superstitious"
+    - "placeholder"
+
+# Settings for clearing mode
+clearing:
```

Comment:
should have a deletion mode, where instead of just closing unlucky numbers, it deletes them. Should default to false